### ISSUE TITLE
Cannot connect to MySQL databases through SSH tunneling

### ISSUE THREAD

**Issue by MuresanSergiu** [kndndrj/nvim-dbee#162]
I have various connection to databases using SSH tunneling but dbee doesn't seem to be able to connect to them.

If I use this connection string:
"http://root:pass@tcp(127.0.0.1:5000)/database"
It fails with error
    c.db.Conn: Error 1045 (28000): Access denied for user 'root'@'localhost' (using password: YES)
It somehow changed the 127.0.0.1 to localhost which is not correct.

But if I change it to
"http://root:pass@127.0.0.1:5000/database"
It fails with error
adapters.NewConnection: core.NewConnection: adapter.Connect: unable to connect to mysql database: default addr for network '127.0.0.1:5000' unknown

There's nothing that I can do to make this work therefore I cannot use this plugin...

**Comment by caliguIa** [comment]
also having this issue

**Comment by alexrah** [comment]
thanks for sharing. Inspecting your configuration I don't understand how a SSH tunnel created with command `ssh -N ous-staging` without port forwarding can route traffic from local nvim to remote SQL server. 
I can only think about setting up `LocalForward` in `~/.ssh/config` but maybe there's another way

**Comment by caliguIa** [comment]
Yeah so in my `~/.ssh/config` I have the following as an example:

````
Host ous-staging
  User $username
  HostName $hostname
  Port $port
  IdentityFile $pathToPemFile
  LocalForward $port $url:$remotePort

````

Actual values obfuscated of course. Let me know if you have any other questions, happy to help!

**Comment by moeezali2375** [comment]
i cant even connect to my local mysql same error is there. is there any workaround?

**Comment by aradores** [comment]
> i cant even connect to my local mysql same error is there. is there any workaround?

Mine too. 

```
{"id": "file_source_/MkxkOOHlAy", "url": "mysql://root:secretpassword@localhost:3307/ayasib", "name": "mysql", "type": "mysql"}]
```

This connection should be working.
